pkg/processor/eventsource/http/eventsource.go

```diff
@@ -120,6 +120,8 @@ func (h *http) requestHandler(ctx *fasthttp.RequestCtx) {
 
 		// set the function logger to that of the chosen buffer logger
 		functionLogger, _ = nucliozap.NewMuxLogger(bufferLogger.Logger, h.Logger)
+	} else {
```

Comment:
functionLogger is an _override_ logger. I specifically didn't want to pass the function logger each time - if it's nil it tells the downstream chain to use whatever it was given during initialization as a function logger. I don't want to have to provide this each time since event sources don't necessarily have access to whatever was given to its runtimes, workers, etc (it's out of their scope of event source responsibility).

Reply:
OK